spec/lib/vmdb/loggers/multicast_logger_spec.rb

```diff
@@ -26,4 +26,13 @@
     subject.level = 3
     [logger1, logger2, subject].each { |l| expect(l.level).to eq(3) }
   end
+
+  context "#<<" do
+    it "forwards to the other loggers" do
+      expect(logger1).to receive(:<<).with("test message")
+      expect(logger2).to receive(:<<).with("test message")
```

Comment:
I know this has been the standard with these tests, but can we possibly just test that the message was written to the file, and not that we expect it to receive a message:

```ruby
# should probably just make the `StringIO`'s their own `let()` if you decide to do this,
# but putting this all in one place for now for simplicity
it "forwards to the other loggers" do
  logfile1 = logger1.instance_variable_get(:@logdev).dev
  logfile2 = logger2.instance_variable_get(:@logdev).dev
  subject << "test #<< message"

  expect(logfile1.tap(:&rewind).read.lines.last).to eq("test #<< message")
  expect(logfile2.tap(:&rewind).read.lines.last).to eq("test #<< message")
end
```

I personally try to test without using stubs whenever possible, and it seems like this is a place we could do that without too much trouble.